src/scilpy/cli/scil_bundle_score_many_bundles_one_tractogram.py

```diff
@@ -7,30 +7,63 @@
+- scil_bundle_pairwise_comparison:
+    - can compare any pair (any combo) from the input list.
+    - can compare the input list against a single file (--single_compare)
+- scil_bundle_score_many_bundles_one_tractogram:
+    - scores files against their individual ground truth.
+- scil_bundle_score_same_bundle_many_segmentations:
+    - compare many versions of a single bundle.
+* If you have volumes associated to your bundles, the following script could be
+of interest for you: scil_volume_pairwise_comparison
+
+This script
+-----------
+We expect your tractogram to be already segmented into valid bundles.
+See our scripts scil_tractogram_segment_with [...].
+For usage in combination with scil_tractogram_segment_with_ROI_and_score, see
+particular instructions below.
 
-This script is the second part of script scil_score_tractogram, which also
-segments the wholebrain tractogram into bundles first.
+The easiest usage of this script requires:
+    - Having all bundles to be scored in a single folder.
+    - Creating a config file associating each input tractogram to its
+      reference tractogram.
+>> scil_bundle_score_many_bundles_one_tractogram config_file VB_folder
 
-Here we suppose that the bundles are already segmented and saved as follows:
-    main_dir/
-        segmented_VB/*_VS.trk.
-        segmented_IB/*_*_IC.trk   (optional)
-        segmented_WPC/*_wpc.trk  (optional)
-        IS.trk  OR  NC.trk  (if segmented_IB is present)
+Additionnaly, this script can include, in the output json, the percentage of
+invalid streamlines (IS) or invalid bundles, using --is, --ib, etc.
 
 Config file
 -----------
 The config file needs to be a json containing a dict of the ground-truth
-bundles as keys. The value for each bundle is itself a dictionnary with:
-
-    - gt_mask: expected result. OL and OR metrics will be computed from this.*
+bundles as keys. The value for each bundle is itself a dictionnary with the
+path to reference bundles. The refernce names must be included in the
+segmented bundles' names. Ex: VB_folder/segmented_CC_subjX.trk.
 
+Example config file:
+{
+  "CC": "PATH/ref_grouth_truth_CC.nii.gz",
+  "OR_L": "PATH/ref_grouth_truth_OR_L.nii.gz",
+}
 * Files must be .tck, .trk, .nii or .nii.gz. If it is a tractogram, a mask will
 be created. If it is a nifti file, it will be considered to be a mask.
 
-Exemple config file:
+Usage as part 2 of segment_with_ROI
+-----------------------------------
+This script can be used as the second part of script
+>> scil_tractogram_segment_with_ROI_and_score
+Then, we suppose that the bundles are already segmented and saved as follows:
+    root_dir/
+        segmented_VB/*_VS.trk.
+        segmented_IB/*_*_IC.trk   (optional)
+        segmented_WPC/*_wpc.trk  (optional)
+        IS.trk  OR  NC.trk  (if segmented_IB is present)
+Use option --part2_ROI_segmentation, and, as main input folder, give the
+args.root_dir instead of the VB_folder. We will automatically set VB_folder, --is,
```

Comment:
I would have said root_dir instead of args.root_dir. I was about to look for a root_dir arg in this script.